### ISSUE TITLE
manage delay server

### ISSUE THREAD

**Issue by trel** [irods/irods_client_zone_management_tool#141]
- see delay server status
- manage host of delay server
- list jobs
- delete jobs

**Comment by ganning127** [comment]
After speaking with Kory, we believe it is only possible right now to do the equivalent of `iadmin set_delay_server` on ZMT. `get_delay_server` won't work due to the cpp API needing `action` and `target`. I'll implement the set_delay_server command on ZMT as a new page for eventually when more features are able to be added.

**Comment by trel** [comment]
they're just calls within the general admin?   `get_delay_server_info` won't work?

**Comment by korydraughn** [comment]
No. That requires a call to `rc_get_grid_configuration_value()` which isn't exposed via the rest api.

**Comment by trel** [comment]
ah, b/c it's not just for admins...
